packages/core/content-releases/server/src/services/release.ts

```diff
@@ -0,0 +1,34 @@
+import { setCreatorFields } from '@strapi/utils';
+import type { LoadedStrapi } from '@strapi/types';
+import { RELEASE_MODEL_UID } from '../constants';
+import type { ReleaseData, UserInfo } from '../types';
+
+const createReleaseService = ({ strapi }: { strapi: LoadedStrapi }) => ({
+  async create(releaseData: ReleaseData, { user }: { user: UserInfo }) {
+    const releaseWithCreatorFields = await setCreatorFields({ user })(releaseData);
+
+    const release = await strapi.entityService.create(RELEASE_MODEL_UID, {
+      data: releaseWithCreatorFields,
+    });
+
+    return release;
+  },
+  async findMany(query: Record<string, unknown>) {
+    const { results, pagination } = await strapi.entityService.findPage(RELEASE_MODEL_UID, {
+      ...query,
+      populate: {
+        actions: {
+          // @ts-expect-error TS error on populate, is not considering count
+          count: true,
+        },
+      },
+    });
+
+    return {
+      data: results,
```

Comment:
Weren't we also going to return a meta object with:

```
meta: {
	// Derived from releasedAt
	status: 'planned' | 'released'
}
```

Do we still want to do that or do you think it's not necessary?

Reply:
I'm not sure, do we need this at some point on the server? 🤔 This is the kind of calculation I prefer to left to the client, or wdyt?

Reply:
Let's leave it out for now and we can address it when we need it :)